src/k8s-extension/azext_k8s_extension/_help.py

```diff
@@ -15,13 +15,19 @@
 
 helps[f'{consts.EXTENSION_NAME} create'] = f"""
     type: command
-    short-summary: Create a Kubernetes Extension.
+    short-summary: Create a Kubernetes Cluster Extension, including purchasing a Kubernetes Offer from Azure Marketplace. Please refer to the example at the end to see how to create an extension or purchase an extension offer.
```

Comment:
Extension offer from Market Place is available only for AKS clusters, right?  If so, I think it'll be good to include the text 'AKS only' in parenthesis in the help text.

Reply:
specified AKS only